staking/programs/staking/src/context.rs

```diff
@@ -208,7 +209,7 @@ pub struct CreatePosition<'info> {
         mut,
         seeds = [TARGET_SEED.as_bytes(),&target_with_parameters.get_target().get_seed()[..]],
```

Comment:
almost feels like this seed should be hardcoded to be the governance seed

Reply:
Done